lib/screens/v2/widget_instance/departures.ex

```diff
@@ -259,18 +264,24 @@ defmodule Screens.V2.WidgetInstance.Departures do
   end
 
   @doc """
-  Groups all departures of the same route and headsign, limiting each group to `max_entries_per_group` entries.
+  Groups all departures of the same route and headsign.
+  If a group's first departure is less than 2 minutes away, read two departures, otherwise only read the first departure.
   `notice` rows are never grouped.
 
   The list is ordered by the occurrence of the _first_ departure of each group--later departures can "leap frog"
   ahead of other ones of a different route/headsign if there's an earlier departure of the same route/headsign.
   """
-  @spec group_all_departures(list(Departure.t() | notice), integer) ::
+  @spec group_all_departures(
+          list(Departure.t() | notice),
+          DateTime.t(),
+          integer(),
+          Screen.app_id()
+        ) ::
           list(
             {:normal, list(Departure.t())}
             | {:notice, notice}
           )
-  def group_all_departures(departures, max_entries_per_group) do
+  def group_all_departures(departures, now, max_entries_per_group, app_id) do
```

Comment:
* Do we really need `max_entries_per_group` as a parameter here? In the one place this function is called, the value is hard-coded to `2`. So it seems like we could just push the hard-coding a level down and not have the parameter. (we have `Enum.take(departure_group, 2)` below, and that `2` is hard-coded, which I think is fine)
* As far as I can tell, this could be a `defp`; nothing else calls it.

Reply:
Yeah I suppose it's a bit unnecessary.